functions/src/index.js

```diff
@@ -1,29 +1,65 @@
 import shuffle from 'lodash/shuffle';
-const functions = require('firebase-functions');
+import sortBy from 'lodash/sortBy';
+import { firestore } from 'firebase-functions';
+
+const assignCardsToPlayer = (cards, player) => ({ ...player, cards });
+
+const documentAllTable = firestore.document('tables/{tableId}');
```

Comment:
i don't understand this name it confuses me more than ...

Reply:
Fair enough